.github/workflows/releases.yml

```diff
@@ -76,20 +82,27 @@ jobs:
           github_event_name: ${{ github.event_name }}
           github_action_name: ${{ github.event.action}}
 
+      - name: Setup application version
+        run: |
+          echo "$( jq '.+{DataAggregatorVersion:"${{ steps.get_release.outputs.tag_name }}"}' src/DataAggregator/appsettings.json )" > src/DataAggregator/appsettings.json
+          echo "$( jq '.+{GatewayApiVersion:"${{ steps.get_release.outputs.tag_name }}"}' src/GatewayAPI/appsettings.json )" > src/GatewayAPI/appsettings.json
+
       - name: Docker build and push DataAggregator
         uses: docker/build-push-action@v2
         with:
           file: ./src/DataAggregator/Dockerfile
-          TAGS: |
+          tags: |
             ${{ steps.setup_tags.outputs.data_aggregator_tags }}
           push: true
+          platforms: linux/amd64,linux/arm64
           context: ./
 
       - name: Docker build and push Gateway API
         uses: docker/build-push-action@v2
         with:
           file: ./src/GatewayAPI/Dockerfile
-          TAGS: |
+          tags: |
             ${{ steps.setup_tags.outputs.gateway_api_tags }}
           push: true
+          platforms: linux/amd64,linux/arm64
```

Comment:
If I understand it correctly, this part is only about building the docker image. Was the application built with linux/arm64 support? I see only `--runtime linux-x64`.

Reply:
Is it necessary to build the application two times in the pipeline? If I understood it correctly it is being built during build and publish-docker?

Reply:
Should be enough according to the docs and to what I've tested locally using `docker buildx` - tool used under the hood by this workflow step.

When it comes to building: is it necessary? if we want to keep producing docker container and ZIP-archived release then short answer would be: yes, it is.

Reply:
I was wondering if the docker job could read the artifact created in the build job.